docs/setting_region.md

```diff
@@ -13,27 +13,41 @@ are located in the same region:
 * Your pipeline's temporary location set by `--temp_location` flag.
 * Your output BigQuery dataset set by `--output_table` flag.
 * Your Dataflow pipeline set by `--region` flag.
+* Your Life Sciences API location set by `--location` flag.
 
 ## Running jobs in a particular region
 The Dataflow API [requires](https://cloud.google.com/dataflow/docs/guides/specifying-exec-params#configuring-pipelineoptions-for-execution-on-the-cloud-dataflow-service)
 setting a [GCP
 region](https://cloud.google.com/compute/docs/regions-zones/#available) via
-`--region` flag to run. In addition to this requirment you might also
+`--region` flag to run.
+
+When running from Docker, the Cloud Life Sciences API is used to spin up a
+worker that launches and monitors the Dataflow job. Cloud Life Sciences API
+is a [regionalized service](https://cloud.google.com/life-sciences/docs/concepts/locations)
+that runs in multiple regions. This is set with the `--location` flag. The
+Life Sciences API location is where metadata about the pipeline's progress
+will be stored, and can be different from the region where the data is
+processed. Note that Cloud Life Sciences API is not available in all regions,
+and if this flag is left out, the metadata will be stored in us-central1.
```

Comment:
Let's include:
"See the list of [Currently Available Locations](https://cloud.google.com/life-sciences/docs/concepts/locations)"

Reply:
Added